fabric8-maven-plugin-build/src/test/resources/spring-boot-http-booster/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2016-2017 Red Hat, Inc, and individual contributors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.openshift.booster</groupId>
+    <artifactId>spring-boot-booster-parent</artifactId>
+    <version>1.5.10-15</version>
+  </parent>
+
+  <artifactId>spring-boot-rest-http</artifactId>
+  <version>1.5.10-5-SNAPSHOT</version>
+
+  <name>Spring Boot - HTTP Booster</name>
+  <description>Booster for FMP Integration Test with Cube</description>
+
+  <properties>
+    <arquillian-cube.version>1.15.3-SNAPSHOT</arquillian-cube.version>
```

Comment:
Did you try potential fixes we discussed?

Reply:
Yes I tried. Currently we are setting `arquillian-cube.version` from failsafe plugin https://github.com/arquillian/arquillian-cube/pull/1013/commits/9fff3558f204a81159e3784f35589255a7cca94c#diff-0c972d5eaf7f2ea01468a7f9162f3ae5R52, which we are passing it to embedded maven build from [test](https://github.com/arquillian/arquillian-cube/pull/1013/commits/9fff3558f204a81159e3784f35589255a7cca94c#diff-519e7e2fb47df75b6fe949bceb6c7687R39). So we are good for maven build.

Reply:
But then we don't really need this to be set up here or we set it to be released version (soon to be :)) so it will work regardless. I think I like the second version more. Then we have to update the comment in the test that only if you want to run it from the IDE against latest code changes you have to change it.

Reply:
I think we don't need this version to be set in this project as we have default value to `1.15.3-SNAPSHOT` in [test](https://github.com/arquillian/arquillian-cube/pull/1013/commits/9fff3558f204a81159e3784f35589255a7cca94c#diff-519e7e2fb47df75b6fe949bceb6c7687R39). So I can change it to `1.15.3` and update this property in pom.xml to `<arquillian-cube.version/>